packages/framework/src/Component/FileUpload/FileUpload.php

```diff
@@ -51,7 +48,7 @@ class FileUpload
     protected $filesystem;
 
     /**
-     * @var \Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface|null
+     * @var \Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface
      */
     protected $parameterBag;
```

Comment:
I think its safe to set explicit type here and for all other properties. But it just nice to have in this PR.
```suggestion
    protected ParameterBagInterface $parameterBag;
```

Reply:
please let's move typed properties into a separate PR